tests/e2e/Employee.test.ts

```diff
@@ -0,0 +1,102 @@
+import { test, Page, expect } from "@playwright/test";
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  page = await browser.newPage();
+
+  // Click the Confirm button in the dialog that appears when the request is successful.
+  page.on("dialog", (dialog) => dialog.accept());
+
+  await page.goto("/employees");
+  await page.waitForLoadState("networkidle");
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe("Employee page", () => {
+  test("Reading employee", async () => {
+    expect(
+      await page.getByRole("table").getByRole("row").allTextContents()
+    ).toStrictEqual([
+      // table header
+      "IDName",
+      // table contents
+      "1foo",
+      "2bar",
+      "3baz",
+      "4error employee",
+    ]);
+  });
+
+  test("Creating employee", async () => {
+    await page.locator("text=/Create new employee/i").click();
+    await expect(page).toHaveURL(/.*new/);
+
+    await page.locator("data-testid=input-name").fill("foobar");
+
+    await Promise.all([
+      page.waitForResponse((res) => res.url().includes("/employee")),
+      page.locator("input[type=submit]").click(),
+    ]);
+
+    // Make sure the updated values are reflected after the page is reloaded
+    await page.reload();
+    await expect(page.locator('text="foobar"')).toBeVisible();
+  });
+
+  test("Updating employee", async () => {
```

Comment:
When I added `test.only` to this test case, it would fail because this test case depends on test case above.
I think we should write test cases which are isolated and can be run independently.

Reply:
> I think we should write test cases which are isolated and can be run independently.

For sure. I agree with you 👍🏼

Done in https://github.com/monstar-lab-oss/reactjs-boilerplate/pull/238/commits/c1a739d80ba99b00b60cf6baa1c6d9205db0e5b9